ISSUE_TEMPLATE/testing.md

```diff
@@ -60,19 +59,27 @@ upgrade` will have the new update.
 In the future, the signing part will be integrated in the build job and the OSTree commit import will be integrated in the release job.
 
 - [ ] Open an issue on https://pagure.io/releng similar to https://pagure.io/releng/issue/8578 to ask for the artifacts to be signed and OSTree commit to be imported
+- [ ] Post a link to the issue as a comment in this issue
 - [ ] Wait for releng to process the request
 - [ ] Verify that the image artifact signatures are present and have the right ACL, e.g.:
 
 ```
 aws s3 ls --recursive s3://fcos-builds/prod/streams/testing/builds/$VERSION/
 curl -I https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/$VERSION/x86_64/fedora-coreos-$VERSION-qemu.qcow2.xz.sig
+```
+
+  If the ACL for the signatures are wrong, here's a gnarly one-liner to fix them all:
+
+```
+aws s3api list-objects --bucket fcos-builds --prefix prod/streams/testing/builds/$VERSION --output json | jq '.Contents[].Key' -r | grep '.sig$' | xargs -n1 aws s3api put-object-acl --bucket fcos-builds --acl public-read --key
```

Comment:
I wish we knew what was causing this. Do you think the permissions mohan has aren't enough?

Reply:
I'm not sure, but thankfully we'll (hopefully) only have to go through this once or twice more.